specification/monitor/resource-manager/readme.md

```diff
@@ -35,13 +35,21 @@ title: MonitorManagementClient
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2022-06
-
+tag: package-2022-07
```

Comment:
Hi @avjerafi , is the 'package-2022-07' correct tag? I don't see it's definition in this readme file

Reply:
@changlong-liu That was not correct. Pushed a fix.